### ISSUE TITLE
Catalan translation

### ISSUE THREAD

**Issue by darmng** [ncase/anxiety#34]
Collaborators welcome.

https://github.com/darmng/anxiety

I'm going to start working on the translation this winter. If anybody wants to collaborate please reply to this issue and @mention me and we'll get organized.

Thanks

**Comment by miquelangelfuentes** [comment]
Hi!

Count on me, @darmng! 

I'm also on Twitter (@MAFuentesArjona). Feel free to get in touch!

**Comment by darmng** [comment]
Thanks @miquelangelfuentes ! I'll get in touch. I want to start the translation this December ^^

**Comment by miquelangelfuentes** [comment]
Yeah, that'd be a good time since I'll be on holiday break. Do not hesitate to get in touch with me!

**Comment by darmng** [comment]
Hi. I think it's pretty obvious, but I haven't been able to start the translation and I won't be doing it for the next few months.

I hope someone else can do it, because it's an interesting game to be accessible to as many people as possible.

**Comment by CatalonianPunIntended** [comment]
I'm doing a translation. It has been in progress for 6 months. It only needs graphical assets (which I'm doing) and some checking. Any contributions are welcome, of course.

https://github.com/CatalonianPunIntended/anxiety

**Comment by CatalonianPunIntended** [comment]
Some months ago I finished it. I sent an e-Mail to the developer, hope they sees it.
